go.mod

```diff
@@ -246,5 +246,7 @@ replace (
 	//  broken goleveldb
 	github.com/syndtr/goleveldb => github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
 
+	github.com/tendermint/tendermint => github.com/cometbft/cometbft v0.34.29
```

Comment:
not needed, we don't have `github.com/tendermint/tendermint` imports

Reply:
hmm weird, for me, and I guess for @zale144 , goland complains if this isnt there

but it does seem possible to build without it

Reply:
try running the command `go list -m all` and you'll see an error

Reply:
should be fixed by
`github.com/osmosis-labs/osmosis/osmomath => github.com/dymensionxyz/osmosis/osmomath v0.0.6-dymension-v0.1.0.20240820121212-c0e21fa21e43`

Reply:
done